### ISSUE TITLE
Review of the lab2 solution - done by Francesca Zafonte

### ISSUE THREAD

**Issue by Zafonte** [AmorosoLaura/computational_intelligence#3]
Hi Laura, I reviewed your code and I'm glad to share my feedback.

**_README:_**
The README file is well written and very explanatory of the problem.
I just suggest you add some more information about how you decided to implement the evolutionary strategy.

**_CODE:_**
The code is well formatted and easily readable. The names of variables, functions and comments are descriptive.
In general the problem was carried out very well and there are not many things to say.

- Task 2.1
Congratulations for implementing a better solution to task 2.1. I really appreciated it.

- Task 2.2
I found that your fitness function contains repetitive code that you can simplify. For example, you can create a list of strategies and use a loop to go through them.
About the ES, I like your implementation and I found it very clarify. 

To summarize, I think that your solution to the problem was implemented really well and I hope that you'll find my little tips useful!
